new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1155 @@
+
+     :Vimtutor <Enter>
+
+  4. If you have these steps memorized and are confident, execute steps
+     1 through 3 to exit and re-enter the editor.
+
+NOTE:  The command  [:q!](:q) <Enter>  discards any changes you have made. In a
+       few lessons you will learn how to save the changes to a file.
+
+  5. Move the cursor down to Lesson 1.3.
+
+# Part 1: COUNT, OPERATOR, MOTION -- ACTION
+
+Before we continue we need to understand the concept of modal editing first:
+most editors have two "modes", a mode where you enter text and a mode where you
+select text.
+
+You have already met two of Vim's modes so far: normal mode which
+is the default mode of Vim where you move around the cursor, and command mode
+when you entered the command to quit Vim. You will soon meet insert mode which
+lets you type text and visual mode which lets you visually select text. Out of
+these normal mode is where you will spend most of your time.
+
+Keyboard keys have different effects depending on the mode you are in: pressing
+the «j» key in normal mode will move the cursor downwards, but pressing the
+same key in insert mode will place a 'j' character in your text.
+
+The next important thing we need to keep in mind is how the syntax of Vim
+commands works: unlike most editor where keyboard commands are just shortcuts,
+Vim commands are part of a language with its own grammar and vocabulary.
+
+The structure of a "sentence" is *count* *operator* *motion*. Don't think too
+much about if for now, all that is important to know is that once you have
+learned a new vocable you can use it along with the ones you already know to
+construct completely new sentences. This is what makes Vim so productive, you
+tell it exactly what it is you want to do. Soon a sequence like this
+
+    <Esc>0f<SPACE>d3wP$P
+
+will read to you like a regular sentence instead of a seemingly random
+collection of letters and symbols.
+
+You have already learned four motions so far: h, j, k and l.
+
+
+# Lesson 2.4: USING A COUNT FOR A MOTION
+
+** Typing a number before a motion repeats it that many times. **
+
+  1. Move the cursor to the start of the line marked ---> below.
+
+  2. Type  «2l»  to move the cursor two characters forward.
+
+  3. Type  «3h»  to move the cursor three character backwards.
+
+  4. Type  «0»  ([zero](0)) to move to the start of the line.
+
+  5. Repeat steps 2 and 3 with different numbers.
+
+---> This is just a line with words you can move around in. {expect:NULL}
+
+  6. Move on to Lesson 2.5.
+
+Strictly speaking 0 on its own is not a count, it is a motion that moves to the
+beginning of the current line, but a count can contain the digit 0. Typing «10j»
+will move the cursor ten lines down.
+
+You can now move more than one character at a time, but constantly typing in a
+count is cumbersome. There are better motions for moving around.
+
+# Lesson something.something: WORD-WISE MOTION
+
+** Moving word-wise is faster than character-wise **
```

Comment:
I would still suggest a more neutral tone. In context, this is a value judgement, and it's kinda dumb :p (we should not be less smart than our readers) I would be happier just by replacing the "is" with "can be". Then the lesson teaches *when* it can.